packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -233,6 +237,13 @@ export class RafikiService implements IRafikiService {
       { status: 'COMPLETED', value: amount.value }
     )
 
+    const user = await this.deps.userService.getByWalletId(receiverWalletId)
+    if (user)
+      await this.deps.socketService.emitMoneyReceivedByUserId(
+        user.id.toString(),
+        amount.assetCode
```

Comment:
We should send the full amount object, so we can specify how much money the user received.